torch/nn/modules/channelshuffle.py

```diff
@@ -9,38 +9,35 @@ class ChannelShuffle(Module):
     r"""Divides and rearranges the channels in a tensor.
 
     This operation divides the channels in a tensor of shape :math:`(*, C , H, W)`
-    into g groups and rearranges them as :math:`(*, \frac{C}{g}, g, H, W)`,
-    while keeping the original tensor shape.
+    into g groups as :math:`(*, C \frac g, g, H, W)` and rearranges them,
```

Comment:
The rendering here seems wrong

https://docs-preview.pytorch.org/pytorch/pytorch/123959/generated/torch.nn.ChannelShuffle.html#channelshuffle
<img width="992" alt="Screenshot 2024-04-12 at 5 39 26 PM" src="https://github.com/pytorch/pytorch/assets/35276741/804c443f-b915-4acd-ac09-db11b6199094">

Reply:
Fixed it in [8abfdf8](https://github.com/pytorch/pytorch/pull/123959/commits/8abfdf8120411f67aa2375af30b45e807baab870) and it seems to render correct locally now:
<img width="788" alt="Screenshot 2024-04-14 at 4 19 06 PM" src="https://github.com/pytorch/pytorch/assets/26398753/2a0a1174-5cc7-46ef-aae0-c06cd9c781fa">